Sources/Lumina/Camera/Extensions/CameraActionsExtension.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CameraActionsExtension.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+extension LuminaCamera {
+  func getPreviewLayer() -> AVCaptureVideoPreviewLayer? {
+    let previewLayer = AVCaptureVideoPreviewLayer(session: self.session)
+    previewLayer.videoGravity = AVLayerVideoGravity.resizeAspectFill
+    return previewLayer
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/CameraActionsExtension.swift#L18"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>CameraActionsExtension.swift:18</code></td>
    </tr>
  </tbody>
</table>